src/lib/registration/registrationInitialisation.ts

```diff
@@ -0,0 +1,81 @@
+import { BeaconUse, IRegistration } from "./types";
+
+/**
+ * Convenience function for returning an `empty` instance of a beacon registration.
+ *
+ * @returns {IRegistration}   JSON representation of a beacon registration
+ */
+export const initBeacon = (): IRegistration => {
+  return {
+    manufacturer: "",
+    model: "",
+    hexId: "",
+
+    manufacturerSerialNumber: "",
+    chkCode: "",
+    batteryExpiryDate: "",
+    batteryExpiryDateMonth: "",
+    batteryExpiryDateYear: "",
+    lastServicedDate: "",
+    lastServicedDateMonth: "",
+    lastServicedDateYear: "",
+
+    ownerFullName: "",
+    ownerEmail: "",
+    ownerTelephoneNumber: "",
+    ownerAlternativeTelephoneNumber: "",
+    ownerAddressLine1: "",
+    ownerAddressLine2: "",
+    ownerTownOrCity: "",
+    ownerCounty: "",
+    ownerPostcode: "",
+
+    emergencyContact1FullName: "",
+    emergencyContact1TelephoneNumber: "",
+    emergencyContact1AlternativeTelephoneNumber: "",
+    emergencyContact2FullName: "",
+    emergencyContact2TelephoneNumber: "",
+    emergencyContact2AlternativeTelephoneNumber: "",
+    emergencyContact3FullName: "",
+    emergencyContact3TelephoneNumber: "",
+    emergencyContact3AlternativeTelephoneNumber: "",
+
+    uses: [initBeaconUse()],
+  };
+};
+
+/**
+ * Convenience function for returning an `empty` instance of a beacon use.
+ *
+ * @returns {BeaconUse}   JSON representation of a beacon use
+ */
+export const initBeaconUse = (): BeaconUse => {
```

Comment:
Idea is that we are not concerned with, is the user submitting a use for:

- Maritime
- Aviation
- Land
- Other

all we care about, at this point within the form, is that we have all the keys that a user could possibly update for a use.  When we're concerned with validating particular fields that are not relevant for a particular use, i.e: callSign for land/other use, we can either:

- Exclude them when we serialise the use to the backend
- Exclude them within the API layer

It made the implementation far easier in reducing the complexity of having to handle scenarios where:

- User has submitted a use case of Maritime
- User goes back to primary use and updates to land
- We now need to update the use to ensure that all fields for land are present on the beacon use json